pkg/scene/widget_align_system.go

```diff
@@ -0,0 +1,250 @@
+package scene
+
+import (
+	"github.com/reearth/reearth-backend/pkg/id"
+)
+
+// WidgetAlignSystem is the layout structure of any enabled widgets that will be displayed over the scene.
+type WidgetAlignSystem struct {
+	inner WidgetZone
+	outer WidgetZone
+}
+
+// WidgetZone is the structure of each layer (inner and outer) of the align system.
+type WidgetZone struct {
+	left   WidgetSection
+	center WidgetSection
+	right  WidgetSection
+}
+
+// WidgetSection is the structure of each section of the align system.
+type WidgetSection struct {
+	top    WidgetArea
+	middle WidgetArea
+	bottom WidgetArea
+}
+
+// WidgetArea has the widgets and alignment information found in each part area of a section.
+type WidgetArea struct {
+	widgetIds []*id.WidgetID
+	align     string
+}
+
+type WidgetLocation struct {
+	Zone    string
+	Section string
+	Area    string
+}
+
+const (
+	AlignStart  = "start"
```

Comment:
```suggestion
	WidgetAlignStart  = "start"
```

Sorry, but I found that these constants should have `Widget` prefix because when we want to use it in other packages, we write `scene.AlignStart` but we may confuse "what is align for???" If they have the prefix, `scene.WidgetAlignStart` is very clear that the "align" is for widgets.

Other constants should be renamed also.

Reply:
Ok